<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateEdit" name="dateAddField">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>d/M/yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dateEnterText">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Date(d/m/y):</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="datesTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>281</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>601</width>
      <height>16777215</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Next Day</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Previous Day</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Is Leap</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Week Number</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Duration</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="dateAddButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>60</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dateEditNumber">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>100</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="dateEditRowTExt">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>80</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Row number:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEditField">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>130</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>d/M/yyyy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dateEditButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QLabel" name="dateBirthdayText">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>180</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Birthday date(d/m):</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateBirthdayField">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>200</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>d/M</string>
    </property>
    <property name="calendarPopup">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="fileText">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>250</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fileOpen_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>270</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fileSave_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>270</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="fileOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="fileSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
